src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -167,6 +169,7 @@ package
         case MediaPlayerState.LOADING:
           break;
         case MediaPlayerState.READY:
+          totalBitratesAvailable();
```

Comment:
Would you mind renaming this function to show what it does?  It provides the bitrate information to the controller